jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PushImageStepTest.java

```diff
@@ -89,6 +98,54 @@ public void testMakeListForManifestList() throws IOException, RegistryException
     }
   }
 
+  @Test
+  public void testMakeList_multiPlatform_enabled() throws IOException, RegistryException {
+    Image asd = Image.builder(V22ManifestTemplate.class).setArchitecture("wasm").build();
+
+    Mockito.when(buildContext.isEnablePlatformTags()).thenReturn(true);
+
+    ImmutableList<PushImageStep> pushImageStepList =
+        PushImageStep.makeList(
+            buildContext,
+            progressDispatcherFactory,
+            registryClient,
+            new BlobDescriptor(mockDescriptorDigest),
+            asd,
+            false);
+
+    ArgumentCaptor<String> tagCatcher = ArgumentCaptor.forClass(String.class);
+    Mockito.when(registryClient.pushManifest(Mockito.any(), tagCatcher.capture())).thenReturn(null);
```

Comment:
Let's do static importing for these:
```suggestion
    when(registryClient.pushManifest(any(), tagCatcher.capture())).thenReturn(null);
```